CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ if( T8CODE_ENABLE_MPI )
     if( NOT MPIEXEC_EXECUTABLE )
         message( FATAL_ERROR "MPIEXEC was not found" )
     endif()
-    set( SC_ENABLE_MPI ON )
+    # set( SC_ENABLE_MPI ON )
```

Comment:
Please add a comment why that is a dirty solution, that it is only temporary till the fixes upstream are available.

Reply:
Done @jmark